src/main/java/frc/robot/subsystems/climber/commands/AdjustAngle.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.climber.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.climber.Climber;
+
+public class AdjustAngle extends CommandBase {
```

Comment:
What is the purpose of this command?

Reply:
IMO we should rename this command to something like `StabilizeClimber`. The purpose of this command is to make the climber horizontal (so we will be in the frame perimeter at the beginning of the match)